whale

```diff
@@ -1,5 +1,7 @@
 docker run \
     -u "$(id -u):$(id -g)" \
     -v $(pwd):/work \
-    -w /work -it -p 3000:3000 \
+    -w /work -it \
+    -p 3000:3000 \ # React port
+    -p 6006:6006 \ # Storybook port
```

Comment:
@Spy13x8 Can you confirm the working status of this script? As far as I understand, retaining these comments were leading to an unexpected behavior.

Reply:
yes my whale script was working totally fine the last time I used it but still I reviewed and corrected it to prevent any future errors